portfolio/src/main/webapp/index.html

```diff
@@ -3,26 +3,121 @@
 
   <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
     <title>Obi's Portfolio</title>
+    <!--Fonts-->
+    <link href="https://fonts.googleapis.com/css2?family=Fira+Code:wght@400;500&family=IBM+Plex+Mono:wght@400;500;700&display=swap" rel="stylesheet">
+    
+    <!--Icons-->
+    <script src="https://kit.fontawesome.com/2e7fd39b04.js" crossorigin="anonymous"></script>
+    <script src="https://code.iconify.design/1/1.0.6/iconify.min.js"></script>
+
+    <!-- Bootstrap CSS only -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+    
+    <!-- custom styles -->
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+
   </head>
 
+
   <body>
-    <div id="content">
-      <!-- Header -->
-      <h1>Obi Abii's Portfolio</h1>
 
-      <!-- Intro Paragraph -->  
-      <p>Hi, My name is Obi and this is my portfolio! <br>
-       I'm a CS Major at Cornell, currently interning at Google for the Summer of 2020 <br>
-       <b>Enjoy!</b></p>
+    <div id="content" class = "container-md">
+      <header>
+        <!-- Navigation bar at the top -->
+        <div class = "row">
+          <nav>
+            <a target = "_blank" href="mailto: coa22@cornell.edu" class="nav_bar">/email_me</a>
+            <a target = "_blank" href="https://github.com/Obinnabii" class="nav_bar">/see_github</a>
+            <a target = "_blank" href="https://drive.google.com/file/d/1tc2YaWmPzVhdCSb3Gv6VVz-599Qa5CrT/view?usp=sharing" class="nav_bar ">/see_resume</a>
+          </nav>
+        </div>
+      </header>
+
+      
 
-       <!-- Greeting generator -->
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <!-- Console -->
+      <div id="console"></div>
 
+      <!--projects -->
+      <!--Space invaders-->
```

Comment:
nit: for all comments, please try to follow the style guide style https://google.github.io/styleguide/htmlcssguide.html and original examples in the file i.e. <!-- (one space after -) (and one space before -) -->